diff --color -ur ympd-1.3.0.orig/src/mpd_client.c ympd-1.3.0/src/mpd_client.c
--- ympd-1.3.0.orig/src/mpd_client.c	2016-02-13 22:01:52.000000000 +0100
+++ ympd-1.3.0/src/mpd_client.c	2022-12-29 16:25:42.882837523 +0100
@@ -27,6 +27,8 @@
 #include "config.h"
 #include "json_encode.h"
 
+struct t_mpd mpd;
+
 /* forward declaration */
 static int mpd_notify_callback(struct mg_connection *c, enum mg_event ev);
 
diff --color -ur ympd-1.3.0.orig/src/mpd_client.h ympd-1.3.0/src/mpd_client.h
--- ympd-1.3.0.orig/src/mpd_client.h	2016-02-13 22:01:52.000000000 +0100
+++ ympd-1.3.0/src/mpd_client.h	2022-12-29 16:25:27.793305437 +0100
@@ -92,7 +92,9 @@
 
     int song_id;
     unsigned queue_version;
-} mpd;
+};
+
+extern struct t_mpd mpd;
 
 struct t_mpd_client_session {
     int song_id;
